token/services/network/fabric/network.go

```diff
@@ -309,3 +319,43 @@ func (n *Network) GetEnrollmentID(raw []byte) (string, error) {
 	}
 	return ai.EnrollmentID(), nil
 }
+
+func (n *Network) requestApproval(context view.Context, namespace string, requestRaw []byte, signer view.Identity, txID driver.TxID) (*fabric.Envelope, error) {
+	return chaincode.NewEndorseView(
+		namespace,
+		InvokeFunction,
+	).WithNetwork(
+		n.n.Name(),
+	).WithChannel(
+		n.ch.Name(),
+	).WithSignerIdentity(
+		signer,
+	).WithTransientEntry(
+		"token_request", requestRaw,
+	).WithTxID(
+		fabric.TxID{
+			Nonce:   txID.Nonce,
+			Creator: txID.Creator,
+		},
+	).Endorse(context)
+}
+
+func (n *Network) requestApprovalFPC(context view.Context, namespace string, requestRaw []byte, signer view.Identity, txID driver.TxID) (*fabric.Envelope, error) {
+	return chaincode.NewEndorseView(
+		namespace,
+		InvokeFunction,
+	).WithNetwork(
+		n.n.Name(),
+	).WithChannel(
+		n.ch.Name(),
+	).WithSignerIdentity(
+		signer,
+	).WithTransientEntry(
+		"token_request", requestRaw,
+	).WithTxID(
+		fabric.TxID{
+			Nonce:   txID.Nonce,
+			Creator: txID.Creator,
+		},
+	).Endorse(context)
```

Comment:
I believe you missed to remove the transient entry and pass the `requestRaw` as parameter :)

Reply:
good catch, fixed :)